pkg/cloud/cloud_test.go

```diff
@@ -82,7 +82,51 @@ func extractVolumeIdentifiers(volumes []types.Volume) (volumeIDs []string, volum
 	}
 	return volumeIDs, volumeNames
 }
+func TestNewCloud(t *testing.T) {
```

Comment:
np: `sucess` misspelled in all these test cases

Reply:
second np: can we put colon after success (like `success: `) for consistency with our test naming elsewhere